server/server.js

```diff
@@ -4,14 +4,16 @@ const Log = require('log');
 
 const db = require('../db/db');
 
+const _ = require('underscore');
+
 const log = new Log();
 
 const app = express();
 const port = process.env.PORT || 3004;
 
 app.use(express.static('client/dist'));
 
-app.get('/projects/:id', (req, res) => {
+app.get('/projects/:ids', (req, res) => {
```

Comment:
If you change this, the request parameters will be different.
It will be req.params.ids, instead of req.params.id 
See line 17

Reply:
changed to id